components/phttp/base.go

```diff
@@ -46,37 +59,45 @@ func (b *Base) Shoot(ctx context.Context, ammo Ammo) {
 	if b.Connect != nil {
 		err := b.Connect(ctx)
 		if err != nil {
-			zap.L().Warn("Connect fail", zap.Error(err))
+			b.Log.Warn("Connect fail", zap.Error(err))
 			return
 		}
 	}
 
 	req, sample := ammo.Request()
+	if b.Config.AutoTag > 0 {
+		sample.AddTag(autotag(b.Config.AutoTag, req.URL))
```

Comment:
Maybe we should only add auto tag if it was not specified in ammo file? Is it simple to do so?

Reply:
Put auto tag configuration under `auto-tag` scope, and add `no-tag-only` option with true by default.